kona-crypto/src/main/java/com/tencent/kona/crypto/provider/SM3Engine.java

```diff
@@ -104,7 +104,8 @@ public void update(byte message) {
         word[wordOffset++] = message;
 
         if (wordOffset >= word.length) {
-            processWord();
+            processWord(word, 0);
+            this.wordOffset = 0;
```

Comment:
I would have mentioned this line: `wordOffset = 0`

Reply:
Sorry for the oversight. I rechecked it, and I guess this commit should be fine now.